tracim/tracim/controllers/content.py

```diff
@@ -216,7 +221,11 @@ def get_one(self, file_id, revision_id=None):
             revision_id = file.revision_id
 
         file_path = content_api.get_one_revision_filepath(revision_id)
-        nb_page = preview_manager.get_nb_page(file_path=file_path)
+        nb_page = 0
+        try:
+            nb_page = preview_manager.get_page_nb(file_path=file_path)
+        except PreviewGeneratorException:
+            pass
```

Comment:
when try/except/pass, it is required to explain why the exception is silenced. Here you must explain that preview_manager is a "bonus" feature, so if something goes wrong, it should not make tracim fail